routes/projectRouter.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const projectDb = require('../data/helpers/projectModel');
+const router = express.Router();
+
+router.post('/', (req, res) => {
+  const project = req.body;
+  projectDb
+    .insert(project)
+    .then(project => {
+      res.status(201).json(project);
+    })
+    .catch(err => {
+      res.status(500).json({
+        error: 'There was an error saving the project to the database',
+      });
+    });
+});
+
+router.get('/', (req, res) => {
+  projectDb
+    .get()
+    .then(projects => {
+      res.status(200).json(projects);
+    })
+    .catch(err => {
+      res.status(500).json(projects);
+    });
+});
+
+router.get('/:id', (req, res) => {
+  const id = req.params.id;
+  projectDb
+    .getProjectActions(id)
```

Comment:
This makes sense for your react app, I think for most applications we would want to get a project by id and also get it's actions.